src/annotator/notebook.js

```diff
@@ -6,8 +6,10 @@ import { createSidebarConfig } from './config/sidebar';
  *
  * @return {HTMLIFrameElement}
  */
-function createNotebookFrame(config) {
+function createNotebookFrame(config, groupId) {
   const sidebarConfig = createSidebarConfig(config);
+  // Explicity set the "focused" group
```

Comment:
As with nearly everything in this set of changes, I have no attachment to doing this in this manner, in any way. It just gets the job done for now...